app/src/main/java/org/hisp/dhis/android/dataentry/form/section/EditTextHintCache.java

```diff
@@ -0,0 +1,7 @@
+package org.hisp.dhis.android.dataentry.form.section;
+
+public interface EditTextHintCache {
```

Comment:
What I propose is to inject prompts into view holder through constructor. It won't really add any complexity to it in any way. It is actually more problematic to have a class with cache which is in my opinion adds quite a bit of confusion.